docker/cli
# WIP: Added context to the error message for 'docker create network' and other similar incorrect patterns (fixes #1986)

**- What I did**
Added additional error message context when user types docker create network instead of docker network create (or config, container, plugin, secret, service, or volume) to point user to the correct command. Fixes issue #1986

Also added additional context to the original error message as suggested by cpuguy83 in the discussion of issue #1986 and fixed a comment typo in create.go

**- How I did it**
Added an additional if statement to the error check in createContainer to note, if a user attempts to create a container named config, container, network, plugin, secret, service, or volume and no container by that name is found, that they may have been attempting to run (for example) 'docker network create' instead of 'docker create network' and points them to the correct command.

Also, as cpuguy83 suggested, I added "Error while creating container" to the beginning of the existing error message to make explicit that the user attempted to create a container (which may not have been their intent).

**- How to verify it**
Type 'docker create network' (or one of the other keywords listed above) to verify the additional context to the error message.

Note that I am having difficulty testing as per the directions for running a development container, so I am not 100% positive this works as intended. Wanted to make sure I was on the right track, however.

**- Description for the changelog**
Added context to the error message for 'docker create network' and other similar incorrect patterns.


**- A picture of a cute animal (not mandatory but encouraged)**
![bear_waving](https://user-images.githubusercontent.com/40665701/60771518-e77d1a00-a0ae-11e9-8286-7b93aaf83aab.gif)

Signed-off-by: Jake Bartolone <jakebartolone@gmail.com>
